jedi/settings.py

```diff
@@ -143,6 +143,11 @@
 ``globals()`` modifications a lot.
 """
 
+allow_descriptor_getattr = True
```

Comment:
Rename this to `instance_allow_descriptor_getattr`.

Reply:
Done